src/design-system/InputDate.css

```diff
@@ -211,7 +211,10 @@
   z-index: 1;
 }
 .react-date-picker__calendar--closed {
-  display: none;
+  opacity: 0;
```

Comment:
this is mysterious to me since the date picker still works in Firefox in other places (e.g. the add cases modal), but if it does fix the issue I don't necessarily have an issue with it. However, one note - because this CSS file is copied out of the package to work around a bug (see comment at the top) I would rather we didn't make the change here, but rather did an override in the styled component inside `DatePicker.tsx` instead.

We might not even need this file anymore because it was an issue specific to Parcel, so I wouldn't want us to cause a regression when we get around to removing it.